backend/slack_commands/execution/session_executor.py

```
@@ -74,7 +75,14 @@ def _execute(
         is_continuation: bool,
         public: bool,
         team_uid: Optional[str] = None,
+        reply_fn=None,
     ) -> None:
+        def _reply(text):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_reply`

Add return type annotation: `None`

(ANN202)